scripts/deploy.sh

```diff
@@ -36,7 +36,12 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
   if [[ ! -d /etc/openstack_deploy/ ]]; then
     ./scripts/bootstrap-aio.sh
     pushd ${RPCD_DIR}
-        for i in $(find etc/openstack_deploy/ -type f -iname '*.yml'); do ../scripts/update-yaml.py /$i $i; done
+      for filename in $(find etc/openstack_deploy/ -type f -iname '*.yml');
+        if [[ ! -a /$filename ]]; then
+          cp $filename /$filename;
+        fi
+        do ../scripts/update-yaml.py /$filename $filename;
```

Comment:
can you properly quote your variables sir?

not needed in the 'if' but everywhere else...
that's something we likely need to do everywhere in this file.